fuzz/proto/db_operation.proto

```diff
@@ -10,7 +10,6 @@ syntax = "proto2";
 
 enum OpType {
   PUT = 0;
-  MERGE = 1;
```

Comment:
Can you explain why it is removed somewhere?

Reply:
just want to keep it simple to start with simple operations like put, delete.